src/components/FeatureViews/Accounts/Accounts.js

```diff
@@ -200,26 +199,19 @@ class Accounts extends Component {
 
                                 </Grid>
                             </TableCell>
-                            <TableCell style={{ width: 100 }}>
-                                {row.user_id}
-                            </TableCell>
-                            <TableCell
-                                style={{ width: 200 }}>
+                            <TableCell>
                                 <Tooltip title={row.email}>
                                     <span>
                                         {row.email.substr(0, 20)}
                                     </span>
                                 </Tooltip>
                             </TableCell>
-                            <TableCell
-                                style={{ width: 100 }}>{addDashes(row.phone_number)}
+                            <TableCell>{addDashes(row.phone_number)}
                             </TableCell>
-                            <TableCell
-                                style={{ width: 150 }}>
+                            <TableCell>
                                 {row.role.charAt(0).toUpperCase() + row.role.slice(1)}
```

Comment:
Use the capitalization function if it is available (may need to merge in development).